experiment_selection.py

```diff
@@ -1,35 +1,39 @@
-import pandas as pd
```

Comment:
These changes are mostly just whitespace due to wrapping them in a page method - I would suggest to look at this PR without whitespace changes